lms/static/scripts/frontend_apps/services/vitalsource.js

```diff
@@ -26,6 +26,25 @@ export class VitalSourceService {
     this._authToken = authToken;
   }
 
+  /**
+   * Fetch a book by bookID ("vbid")
+   *
+   * This is currently a fake that waits for a fixed time before returning
+   * hard-coded data.
+   *
+   * @param {string} bookID
+   * @param {number} [fetchDelay] - Dummy delay to simulate slow third-party
+   * @return {Promise<Book|null>}
```

Comment:
If we're going to use an exception to indicate a failed request then the return type can just be `Promise<Book>`.